src/client/dfuse/il/int_read.c

```diff
@@ -15,22 +15,54 @@
 static ssize_t
 read_bulk(char *buff, size_t len, off_t position, struct fd_entry *entry, int *errcode)
 {
-	daos_size_t read_size = 0;
-	d_iov_t     iov       = {};
-	d_sg_list_t sgl       = {};
-	int         rc;
+	daos_size_t	read_size = 0;
+	d_iov_t		iov       = {};
+	d_sg_list_t	sgl       = {};
+	daos_event_t	ev;
+	bool		flag = false;
+	int		rc;
 
 	DFUSE_TRA_DEBUG(entry->fd_dfsoh, "%#zx-%#zx", position, position + len - 1);
 
 	sgl.sg_nr = 1;
 	d_iov_set(&iov, (void *)buff, len);
 	sgl.sg_iovs = &iov;
-	rc = dfs_read(entry->fd_cont->ioc_dfs, entry->fd_dfsoh, &sgl, position, &read_size, NULL);
+
+	if (daos_handle_is_inval(ioil_eqh))
+		daos_eq_create(&ioil_eqh);
+
+	rc = daos_event_init(&ev, ioil_eqh, NULL);
+	if (rc) {
+		DFUSE_TRA_ERROR(entry->fd_dfsoh, "daos_event_init() failed: %d (%s)",
+				rc, strerror(rc));
```

Comment:
I've not started the migration yet but soon all these calls will be changing to this format:
```suggestion
		DHS_ERROR(entry->fd_dfsoh, rc, "daos_event_init() failed");
```

Reply:
again, i don't care for that in this pr